packages/creator-hub/renderer/src/hooks/useWorkspace.ts

```diff
@@ -73,12 +73,15 @@ export const useWorkspace = () => {
 
   const validateScenesPath = useCallback(async (path: string) => {
     const isDir = await workspacePreload.isDirectory(path);
-    return isDir;
+    const exists = await workspacePreload.exists(path);
+    const isWritable = await workspacePreload.isWritable(path);
+    return isWritable && (isDir || !exists); // Allow non-existing paths as they can be created
```

Comment:
I feel like all this should be a new thunk that calls a validateScenesPath on preload/workspace